node_packages/bazed-svelte/package.json

```diff
@@ -10,7 +10,7 @@
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
     "test:unit": "vitest",
-    "lint": "prettier --plugin-search-dir . --check .",
+    "lint": "eslint .",
```

Comment:
No more checking for adhering to BAZED Ltd.'s styleguide?

Reply:
Fixed!